draft-ppm-protocol.md

```diff
@@ -611,27 +621,31 @@ information specific to the extension.
 
 ## Verifying and Aggregating Reports {#pa-aggregate}
 
-Once a set of clients have uploaded their reports to the leader, the leader can
-send them to the helpers to be verified and aggregated. In order to enable the
-system to handle very large batches of reports, this process can be performed
-incrementally. To aggregate a set of reports, the leader sends an AggregateReq
-to each helper containing those report shares. The helper then processes them
-(verifying the proofs and incorporating their values into the ongoing aggregate)
-and replies to the leader.
-
-The exact structure of the aggregation flow depends on the PPM scheme.
-Specifically:
-
-* Some PPM schemes (e.g., Prio) allow the leader to start aggregating reports
-  proactively before all the reports in a batch are received. Others (e.g.,
-  Hits) require all the reports to be present and must be initiated by the
-  collector.
-
-* Processing the reports -- especially verifying the proofs -- may require
-  multiple round trips.
-
-Note that it is possible to aggregate reports from one batch while reports from
-the next batch are coming in.
+Once a set of clients have uploaded their reports to the leader, the leader and
+helper begin verifying and aggregating them. In order to enable the system to
+handle very large batches of reports, this process can be performed
+incrementally. To aggregate a set of reports, the leader sends a sequence of
+aggregate request to the helper, the first of which contains the helper's
+encrypted input shares. After a number of successful requests, both aggregators
+have recovered a set of valid output shares, which they add into their local
+aggregate output share.
```

Comment:
I think this clearly reveals that these subsequent ones just shouldn't be aggregate requests but something else.

AggregateContinue perhaps.

Reply:
This will be a somewhat major refactor, so I'll wait to address until the next PR.